static/app/utils/analytics/alertsAnalyticsEvents.tsx

```diff
@@ -1,12 +1,31 @@
+import type {MetricRule} from 'sentry/views/alerts/rules/metric/types';
+import type {UptimeMonitorMode} from 'sentry/views/alerts/rules/uptime/types';
+import type {MonitorConfig} from 'sentry/views/insights/crons/types';
+
 export type AlertsEventParameters = {
   'anomaly-detection.feedback-submitted': {
     choice_selected: boolean;
     incident_id: string;
   };
+  'cron_monitor.created': {
+    cron_schedule_type: MonitorConfig['schedule_type'];
+  };
+  'issue_alert_rule.created': Record<string, unknown>;
```

Comment:
I couldn't figure out another typing solution for `issue_alert_rule.created` where there aren't additional parameters